apps/web/lib/plain/plainChat.tsx

```diff
@@ -0,0 +1,266 @@
+              type: "primary",
+            },
+            {
+              icon: "bulb",
+              text: "Send feedback",
+              type: "default",
+            },
+            {
+              icon: "error",
+              text: "Report an issue",
+              type: "default",
+              form: {
+                fields: [
+                  {
+                    type: "dropdown",
+                    placeholder: "Select severity...",
+                    options: [
+                      {
+                        icon: "support",
+                        text: "I'm unable to use the app",
+                        threadDetails: {
+                          severity: "critical",
+                          issueType: "critical",
+                          labelTypeIds: ["lt_01JFJWNWAC464N8DZ6YE71YJRF"],
+                          priority: "u",
+                        },
+                      },
+                      {
+                        icon: "error",
+                        text: "Major functionality degraded",
+                        threadDetails: {
+                          severity: "major",
+                          issueType: "major",
+                          labelTypeIds: ["lt_01JFJWP3KECF1YQES6XF212RFW"],
+                          priority: "h",
+                        },
+                      },
+                      {
+                        icon: "bug",
+                        text: "Minor annoyance",
+                        threadDetails: {
+                          severity: "minor",
+                          issueType: "minor",
+                          labelTypeIds: ["lt_01JFJWPC8ADW0PK28JHMJR6NSS"],
+                          priority: "l",
+                        },
+                      },
+                    ],
+                  },
+                ],
+              },
+            },
+          ],
+          entryPoint: {
+            type: "chat",
+          },
+          hideBranding: true,
+          theme: "auto",
+          style: {
+            brandColor: "#FFFFFF",
+            launcherBackgroundColor: "#262626",
+            launcherIconColor: "#FFFFFF",
+          },
+          position: {
+            bottom: "20px",
+            right: "20px",
+          },
+        };
+
+        if (process.env.NODE_ENV === "development" || process.env.NODE_ENV === "test") {
+          window.__PLAIN_CONFIG__ = plainChatConfig;
+        }
+
+        setConfig(plainChatConfig);
+      } catch (error) {
+        console.error("Failed to initialize Plain Chat:", error);
+      }
+    };
+
+    initConfig();
+  }, [session]);
+
+  const plainChatScript = `
+    window.plainScriptLoaded = function() {
+      if (window.Plain && ${Boolean(config)}) {
+        try {
+          Plain.init(${config ? JSON.stringify(config) : null});
+        } catch (error) {
+          console.error("Failed to initialize Plain:", error);
+        }
+      }
+    }
+  `;
+
+  useEffect(() => {
```

Comment:
This duplicates the hook logic; what do you say about moving this into the hook entirely?